### PR TITLE
Updated GitLeaks to v7.3.0

### PR DESCRIPTION
**Pull request by rseedorff**
## Description

This pull request if applied updates the gitleak version to the latest version v7.3.0.

### Gitleaks Fork
Added a new secureCodeBox gitleaks fork at https://github.com/secureCodeBox/gitleaks

Important! This fork and referenced gitleak image also contains the additional the PR https://github.com/zricethezav/gitleaks/pull/498

### Testing
If you want to test this PR please reference the following docker image in your helmChart: docker pull securecodebox/scanner-git-repo-scanner:git-repo-rate-limit

#### Example HelmChart Values
```yaml
image:
  # image.repository -- Container Image to run the scan
  repository: docker.io/securecodebox/scanner-gitleaks
  # image.tag -- defaults to the charts version
  tag: v7.3.0
```

#### Example HelmChart Install
You can use the following command to install and test this PR in your environment. For more general informations please check the scanner documentation : https://docs.securecodebox.io/docs/scanners/gitleaks
```bash
# Optional: Add the secureCodeBox Helm Repo
helm repo add secureCodeBox https://charts.securecodebox.io
# Install HelmChart (use -n to configure another namespace)
helm upgrade --install gitleaks secureCodeBox/gitleaks \
  --set="image.repository=docker.io/securecodebox/scanner-gitleaks" \
  --set="image.tag=v7.3.0"
```

## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `npm test` runs for the whole project.

### CODE REVIEW

File: scanners/gitleaks/Chart.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ description: A Helm chart for the gitleaks repository scanner that integrates wi
 
 type: application
 # version - gets automatically set to the secureCodeBox release version when the helm charts gets published
-version: v2.5.0-alpha1
-appVersion: v7.3.0
+version: v2.6.0-alpha1
+appVersion: v6.1.2
```
Comment (fuhrmeistery) [inline scanners/gitleaks/Chart.yaml]: Was it intended to downgrade `appVersion` and `version`?

File: scanners/gitleaks/helm2.Chart.yaml [hunk 2]
```diff
@@ -4,8 +4,8 @@ description: A Helm chart for the gitleaks repository scanner that integrates wi
 
 type: application
 # version - gets automatically set to the secureCodeBox release version when the helm charts gets published
-version: v2.5.0-alpha1
-appVersion: v7.3.0
+version: v2.6.0-alpha1
+appVersion: v6.1.2
```
Comment (fuhrmeistery) [inline scanners/gitleaks/helm2.Chart.yaml]: Was it intended to downgrade appVersion and version?

Comment (rseedorff) [reply scanners/gitleaks/helm2.Chart.yaml]: nope 🤔

**Comment by J12934** [comment]
Gitleaks integration tests are failing

**Pull request merged**